### PR TITLE
Update project and branch summary decode

### PR DESCRIPTION
**Pull request by hojberg**
ProjectSummary
--------------
* Decode `owner.handle` expecting a "@" prefix.

BranchSummary
-------------
* Decode the project field using `Project.decodeSummary`.
* Decode `causalHash` expecting a "#" prefix.

### CODE REVIEW

File: src/css/ui/components/anchored-overlay.css [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 /* @color-todo @inverse */
 .anchored-overlay .tag {
-  --c-color_tag_bg : var(--color-gray-darken-20);
```
Comment (hojberg) [inline src/css/ui/components/anchored-overlay.css:33]: unrelated to fix a prettier issue.

**Comment by ChrisPenner** [comment]
Okay, I *think* this is all I needed to change to match this: https://github.com/unisoncomputing/enlil/pull/189

**Review by ChrisPenner — APPROVED** [review]
